### ISSUE TITLE
Remove use of `$.rails` to avoid silent initialization errors

### ISSUE THREAD

**Issue by dcecile** [ifad/data-confirm-modal#61]
_(Continued from https://github.com/twbs/bootstrap-rubygem/issues/127...)_

In integrating `data-confirm-modal` into my Rails app, I accidentally used this ordering in my `application.js` file:

```
//= require rails-ujs
//= require turbolinks
//= require jquery3
//= require popper
//= require bootstrap
//= require_tree .
```

With this ordering `data-confirm-modal` doesn't perform Rails initialization, because `$.rails` wasn't set. I resolved the issue by moving the `//= require jquery3` to the top. With jQuery included first, `data-confirm-modal` initialized without any problems.

I had originally created this as a documentation issue for `bootstrap-rubygem` (which explains the `application.js` setup), but @glebm has suggested that this be resolved by changing `data-confirm-modal` to stop using `$.rails`.

What do you think? Would an alternative to `$.rails` help make the initial setup of `data-confirm-modal` easier?

**Comment by glebm** [comment]
To clarify, in `rails-ujs` this is no longer `$.rails` but `window.Rails`.

You can't rely on `$.rails` because it will be undefined it `rails-ujs` is included before `jquery` (which is possible because `rails-ujs` no longer depends on jQuery).

One option is to use `(window.Rails || $.rails)` instead.

**Comment by vjt** [comment]
Hi, sorry for the delay. I've pushed the sensible, suggested change, that should fix this issue. If all is well I can cut a new release.

**Comment by dcecile** [comment]
I've tried out the fix and it looks good to me.

Before the fix (with `rails-ujs` before `jquery3`), `data-confirm-modal` failed to load, resulting in native modal popups. After the fix, `data-confirm-modal` loaded fine, and I get Bootstrap modal popups. :+1:

**Comment by vjt** [comment]
Released [v1.6.2](https://github.com/ifad/data-confirm-modal/tree/v1.6.2) on [RubyGems](https://rubygems.org/gems/data-confirm-modal/versions/1.6.2) and [npm](https://www.npmjs.com/package/data-confirm-modal).

Thanks for all the feedback!
